src/Chartjs/assets/dist/controller.js

```diff
@@ -34,8 +34,14 @@ class default_1 extends Controller {
     }
     viewValueChanged() {
         if (this.chart) {
-            this.chart.data = this.viewValue.data;
-            this.chart.options = this.viewValue.options;
+            const data = this.viewValue.data;
+            let options = {};
+            if (!Array.isArray(this.viewValue.options)) {
+                options = this.viewValue.options;
+            }
+            this.dispatchEvent('on-view-value-changed', { data, options });
```

Comment:
I'm not sure about this name. 

I feel '`view-value-change`' would be enough, WDYT ?

Reply:
```suggestion
            this.dispatchEvent('view-value-change', { data, options });
```

so the event name will be `chartjs:view-value-change`